test/test_state.py

```diff
@@ -60,9 +64,18 @@ def test_set_and_get_all_attribute(given_that, automation: MockAutomation):
     given_that.state_of(LIGHT).is_set_to('on', attributes={'brightness': 11, 'color': 'blue'})
     assert automation.get_all_attributes_from_light() == {'brightness': 11, 'color': 'blue'}
 
+def test_get_complete_state_dictionary(given_that, automation: MockAutomation):
+    given_that.state_of(LIGHT).is_set_to('on', attributes={'brightness': 11, 'color': 'blue'})
+    given_that.state_of(COVER).is_set_to("closed", {'friendly_name': f"{COVER}", 'current_position': 0})
+    assert automation.get_complete_state_dictionary() == {COVER: {'attributes': {'current_position': 0,
+                                                                                                'friendly_name': COVER},
+                                                                                'state': 'closed'},
+                                                          LIGHT: {'attributes': {'brightness': 11, 'color': 'blue'},
+                                                                                'state': 'on'}}
 
 @pytest.mark.only
 def test_throw_typeerror_when_attributes_arg_not_passed_via_keyword(given_that, automation: MockAutomation):
     given_that.state_of(LIGHT).is_set_to('on', attributes={'brightness': 11, 'color': 'blue'})
+    given_that.state_of(COVER).is_set_to('closed', attributes={'brightness': 11, 'color': 'blue'})
```

Comment:
This is unnecessary. Could you please remove it?

Reply:
You are right, line removed.